core/src/worker/slot_provider.rs

```diff
@@ -42,21 +42,21 @@ impl SlotTrait for Slot {
     }
 }
 
-#[derive(derive_more::DebugCustom)]
+#[derive(derive_more::DebugCustom, Clone)]
```

Comment:
Not sure why this guy actually needs to be clonable now?

Reply:
Replaced cloning with Arc